File: javascript/7_kyu/split_in_parts.js
// Split In Parts
// https://www.codewars.com/kata/5650ab06d11d675371000003/train/javascript

var splitInParts = function(s, partLength){
  var ans = [];
  
  for (var i = 0; i < s.length; i += partLength) {
    ans.push(s.slice(i, i + partLength));
  }
  
  return ans.join(' ')
}

console.log(splitInParts("supercalifragilisticexpialidocious", 3))

File: python/7_kyu/split_in_parts.py
# Split In Parts
# https://www.codewars.com/kata/5650ab06d11d675371000003/train/python

def split_in_parts(s, part_length): 
  ans = []
  for i in range(0, len(s), part_length):
      ans.append(s[i:i + part_length])
  return ' '.join(ans)

print(split_in_parts("supercalifragilisticexpialidocious", 3))